src/integrationTest/java/org/opensearch/test/framework/JwtConfigBuilder.java

```diff
@@ -33,8 +34,8 @@ public JwtConfigBuilder jwtUrlParameter(String jwtUrlParameter) {
         return this;
     }
 
-    public JwtConfigBuilder signingKey(String signingKey) {
```

Comment:
nit: Keep this method as well and have this be overloaded with 2 separate implementations. If a single signingKey is passed then you can create the list in here.